lib/build.js

```diff
@@ -266,10 +285,29 @@ export default {
 
       let stdout = '';
       let stderr = '';
+      let liveErroMatchQueue = '';
+      let liveErroMatchHandle = target.liveErrorMatch ? function (d) {
+        d = liveErroMatchQueue + d;
+        liveErroMatchQueue = '';
+        let index = d.lastIndexOf('\n');
+        if (index >= 0 && index === d.length - 1) {
+          that.errorMatcher.parseAdd(d);
+        } else {
+          liveErroMatchQueue = d.substring(index + 1);
+          that.errorMatcher.parseAdd(d.substring(0, index));
+        }
```

Comment:
That's an actual error output?
Anyway you are right we can't handle all cases. I actually think that impossible.
But you will still handle most of them without breaking anything!
And it won't have weird behaviors.
Refuse it if you want but i am sure i am not the only one who could use that (all interpreted compilation languages could use that)